OneSignalSDK/unittest/src/test/java/com/onesignal/ShadowOSUtils.java

```diff
@@ -32,21 +32,36 @@ public static boolean isGMSInstalledAndEnabled() {
 
    // Huawei: HMS
    public static boolean hasHMSAvailability;
-   public boolean hasHMSAvailability() {
+   public static boolean hasHMSAvailability() {
       return hasHMSAvailability;
    }
    public static boolean hasHMSPushKitLibrary;
```

Comment:
Add a new line above to space out the methods?

Reply:
Agreed, just looking for some space between the end of the getters and the new variables